frontend/public/components/app-contents.tsx

```diff
@@ -192,7 +192,14 @@ const AppContents = withRouter(React.memo(() => (
 
           <LazyRoute path="/k8s/ns/:ns/persistentvolumeclaims/~new/form" exact kind="PersistentVolumeClaim" loader={() => import('./storage/create-pvc' /* webpackChunkName: "create-pvc" */).then(m => m.CreatePVC)} />
 
-          <LazyRoute path="/monitoring" loader={() => import('./monitoring' /* webpackChunkName: "monitoring" */).then(m => m.MonitoringUI)} />
+          <LazyRoute path="/monitoring/query-browser" loader={() => import('./monitoring/metrics' /* webpackChunkName: "monitoring" */).then(m => m.QueryBrowserPage)} />
+          <LazyRoute path="/monitoring/silences/~new" loader={() => import('./monitoring' /* webpackChunkName: "monitoring" */).then(m => m.CreateSilence)} />
+          <LazyRoute path="/monitoring/silences/:id/edit" loader={() => import('./monitoring' /* webpackChunkName: "monitoring" */).then(m => m.EditSilence)} />
+          <LazyRoute path="/monitoring/silences/:id" loader={() => import('./monitoring' /* webpackChunkName: "monitoring" */).then(m => m.SilencesDetailsPage)} />
+          <LazyRoute path="/monitoring/alertrules/:id" loader={() => import('./monitoring' /* webpackChunkName: "monitoring" */).then(m => m.AlertRulesDetailsPage)} />
+          <LazyRoute path="/monitoring/alerts/:ruleID" loader={() => import('./monitoring' /* webpackChunkName: "monitoring" */).then(m => m.AlertsDetailsPage)} />
+          <LazyRoute path="/monitoring" loader={() => import('./alerting' /* webpackChunkName: "alerting" */).then(m => m.AlertingPage)} />
```

Comment:
Actually, this approach doesn't work as is because we want some of these pages to use the poller (e.g. the alert details page).

Reply:
fixed